### PR TITLE
[UI] Remove browse toolbar button and tweak empty gamelist message

### PR DESCRIPTION
**Pull request by Helios747**
This would have been a 20 minute PR but fuck Windows 10's SDK.

image

Fixes blocker: https://bugs.dolphin-emu.org/issues/9500

---
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/3801)

### CODE REVIEW

File: Source/Core/DolphinWX/FrameTools.cpp [hunk 1]
```diff
@@ -544,7 +544,6 @@ void CFrame::PopulateToolbar(wxToolBar* ToolBar)
 
 	WxUtils::AddToolbarButton(ToolBar, wxID_OPEN,               _("Open"),        m_Bitmaps[Toolbar_FileOpen],    _("Open file..."));
 	WxUtils::AddToolbarButton(ToolBar, wxID_REFRESH,            _("Refresh"),     m_Bitmaps[Toolbar_Refresh],     _("Refresh game list"));
-	WxUtils::AddToolbarButton(ToolBar, IDM_BROWSE,              _("Browse"),      m_Bitmaps[Toolbar_Browse],      _("Browse for an ISO directory..."));
```
Comment (degasus) [inline Source/Core/DolphinWX/FrameTools.cpp]: please also remove "IDM_BROWSE", "Toolbar_Browse", and the bitmap itself.

Comment (JosJuice) [inline Source/Core/DolphinWX/FrameTools.cpp]: The bitmap can only be removed if we stop using it in DolphinQt2: https://github.com/dolphin-emu/dolphin/blob/3ff56aa192f952cb6ad2e9c662d4a368efd21e6f/Source/Core/DolphinQt2/ToolBar.cpp#L83

Comment (aserna3) [inline Source/Core/DolphinWX/FrameTools.cpp]: Done

Comment (JosJuice) [inline Source/Core/DolphinWX/FrameTools.cpp]: CFrame::OnBrowse should also be removed.

Comment (aserna3) [inline Source/Core/DolphinWX/FrameTools.cpp]: Done

Comment (JosJuice) [inline Source/Core/DolphinWX/FrameTools.cpp]: Isn't OnBrowse still in Frame.h?

Comment (aserna3) [inline Source/Core/DolphinWX/FrameTools.cpp]: Fixed.

**Comment by JosJuice** [comment]
Reviewed 1 of 2 files at r1, 2 of 4 files at r2, 1 of 1 files at r3, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3801)*

**Comment by JosJuice** [comment]
Reminder: The string freeze is in 41 hours. If this PR's string change isn't merged before then, it can't get into 5.0.

**Comment by MaJoRoesch** [comment]
@delroth are you fine with merging this before 5.0?

**Comment by JosJuice** [comment]
Less than 3 hours left!

**Pull request merged**